### ISSUE TITLE
VM search by IP does not work after the update

### ISSUE THREAD

**Issue by 0sorkon** [apache/cloudstack#9486]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
API?
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.0
~~~

##### SUMMARY

Searching for VMs by IP does not work after upgrading from 4.18.2.1 to 4.19.1.0. Attempting to search by IP gives empty results

##### STEPS TO REPRODUCE

~~~
* Update to 4.19.1.0
* Try to search instance based on IP address
~~~

##### EXPECTED RESULTS

~~~
The search result displays the VM with the searched IP
~~~

##### ACTUAL RESULTS

~~~
Search result is empty.
~~~

**Comment by adietrich-ussignal** [comment]
Can you be a bit more descriptive as to which menu you are attempting to search? Is this from the instance screen where you are attempting to search? 

I've tested on 4.19.1.0 and 4.19.0.2 and in both cases this does not work when done from the Compute > Instances search bar.

**Comment by 0sorkon** [comment]
Yes. I try to search VM from Compute -> Instances page. And it work on 4.18

**Comment by 0sorkon** [comment]
Is there another way to search VMs by IP that works in 4.19?

**Comment by DaanHoogland** [comment]
@0sorkon have you seen this working in older versions? It doesn't sound as familiar functionality to me.

**Comment by 0sorkon** [comment]
Yeah, I'm pretty sure it worked in latest 4.19.

**Comment by weizhouapache** [comment]
this has been fixed by #9547
